docs/node-operators/staking-and-snarking.mdx

```diff
@@ -1,45 +1,64 @@
+You can interact with the Mina network by participating in consensus and helping compress data by generating zk-SNARKs. By operating a node that helps secure the network, you can receive MINA for your efforts. See [Staking Rewards on Mina](https://minaprotocol.com/blog/staking-rewards-on-mina).
 
-:::
+## Participating in Consensus
 
-# Staking & Snarking
+The Mina network is secured by [proof of stake (PoS)](/glossary#proof-of-stake-pos) consensus. With this model of consensus, you don't need to have complex equipment like in Bitcoin mining. By simply having MINA in your wallet, you can choose to stake your MINA or delegate it to another node. 
 
-Let's turn our attention to the other ways we can interact with the Mina network - namely, participating in consensus, and helping compress data by generating zk-SNARKs. By operating a node that helps secure the network, you can receive MINA for your efforts.
+Mina is a public and decentralized blockchain that is open for anyone in the world to participate in actively or passively. You can help increase the security of the network by becoming nodes or block producers, or you can help lower the cost of transactions by becoming SNARK producers, or you can be both.
 
-## Participating in Consensus
+## Staking MINA
+
+You can earn block rewards by participating in block production through staking. Your new stake delegation comes into effect after a latency period of a 2-4 weeks.
 
-The Mina network is secured by [Proof-of-Stake consensus](../glossary#proof-of-stake) With this model of consensus, you don't need to have complex equipment like in Bitcoin mining. By simply having MINA in our wallet, we can choose to either stake it ourselves, or delegate it to another node. Let's first see how to stake MINA ourselves:
+### Requirements
 
-:::caution
+To properly remain synced to the network and participate in consensus, your server must run a [clock synchronization protocol](https://en.wikipedia.org/wiki/Clock_synchronization).
 
-To properly remain synced to the network and participate in consensus, it is important that your server is running some form of a [clock synchronization protocol](https://en.wikipedia.org/wiki/Clock_synchronization). We recommend using [NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol), which is relatively easy to setup and comes already installed as a default service on many popular linux distros.
+Network Time Protocol [(NTP)](https://en.wikipedia.org/wiki/Network_Time_Protocol) is the most common method for synchronizing computer clocks over a network. 
 
-:::
+To synchronize your server software clock with internet time servers, be sure to set up NTP. Many popular Linux distributions include NTP as a default service.
 
-### Staking MINA
 
-We can try out staking with our MINA by issuing the following command:
+To stake with your MINA:
 
-    mina client set-staking --public-key $MINA_PUBLIC_KEY
+```sh
+mina client set-staking --public-key $MINA_PUBLIC_KEY
```

Comment:
protocol team, please verify that the set-staking subcommand is present in the command line help 
(or we can update the doc and instructions if the workflow is changed)